src/base/base.c

```diff
@@ -1067,6 +1101,83 @@ static zend_observer_fcall_handlers xdebug_observer_init(zend_execute_data *exec
 	return (zend_observer_fcall_handlers){xdebug_execute_begin, xdebug_execute_end};
 }
 #endif
+
+static void xdebug_enable_debugger_handlers()
```

Comment:
This function is called when we want to enable the debugger after having been disabled. We restore the function handler and the statement handler. Restoring the function handler in PHP 8.0 is not needed because we won't be disabling it as in PHP 8.0 it is called instead of the Observer API functions